ros_packages/motors/pib_motors/pib_motors/update_bricklet_uids.py

```diff
@@ -65,7 +69,10 @@ def detect_uid_changes():
             update_uids()
         elif methode == "get_uids_from_db":
             get_uids_from_db()
-        elif methode == "detect_uid_changes":
-            detect_uid_changes()
+        elif methode == "no_uids_in_database":
+            no_uids_in_database()
+        elif methode == "check_and_update":
+            if no_uids_in_database():
+                update_uids()
         else:
             print("Method not found")
```

Comment:
Technically speaking `methode` is German and since they don't belong to a class, they are functions.

That being said, instead of using an if statement for every function defined in this file, `globals()` could be used:
```python
try:
    globals()[methode]()
except KeyError:
    print("Method not found")
```